code/modules/mob/living/carbon/alien/status_procs.dm

```diff
@@ -17,4 +17,4 @@
 /mob/living/carbon/alien/AdjustStunned(amount, updating = 1, ignore_canstun = 0)
 	. = ..()
 	if(!.)
-		move_delay_add = min(move_delay_add + round(amount / 2), 10)
+		move_delay_add = min(max(0, move_delay_add + round(amount / 2)), 10)
```

Comment:
Huh, I did this same fix, but used `Clamp()`. Is there any reason you used `max()` instead?